src/app/routes/onDemandRadio/getInitialData/index.test.js

```diff
@@ -46,23 +46,23 @@ describe('Get initial data for on demand radio', () => {
     fetch.mockResponse(
       JSON.stringify(responseWithEpisodeAvailableOneMinuteAgo),
     );
-
     const { pageData } = await getInitialData({
       path: 'mock-on-demand-radio-path',
     });
     expect(pageData.episodeIsAvailable).toEqual(true);
   });
 
-  it('should return episodeIsAvailable as false if current time is before when episode is availableFrom', async () => {
+  it('should return episodeIsAvailable as true if current time is before when episode is availableFrom', async () => {
```

Comment:
This sentence is... well, I had to read it out loud. 

It should return the episode as available if it is consumable in the future?

Reply:
Ahaha yea fair enough. 

Trying to tie the wording to the logic being tested, so opted for "episodeIsAvailable should be true if availableFrom is after current time"